pkg/db/db.go

```diff
@@ -0,0 +1,155 @@
+//Package db provides operations on SQLite DB
+package db
+
+import (
+	"database/sql"
+	"os"
+	// blank import to register sqlite3 driver
+	_ "github.com/mattn/go-sqlite3"
+)
+
+const dbDir string = "/var/lib/leapp/"
+const dbName string = "leapp.db"
+
+//Database type is sql.DB
+type Database sql.DB
+
+//Record type represents DB record description
+type Record struct {
+	UID, Name, Host, DataType, Data string
+}
+
+//EnsureDbPath checks wether the leapp directory exists, if not it tries to create it
+//temporary function, will be solved in packaging later
+func EnsureDbPath() {
```

Comment:
this should not panic but return the error here - And if it really is temporary that should be with the caller directly